ikuzo/driver/elasticsearch/internal/mapping/v2.go

```diff
@@ -125,7 +125,8 @@ var v2Mapping = `{
 						"mimeType": {"type": "keyword"},
 						"periods": {"type": "keyword"},
 						"hasDigitalObject": {"type": "boolean"},
-						"hasRestriction": {"type": "boolean"}
+						"hasRestriction": {"type": "boolean"},
+						"genreform": {"type": "keyword"}
```

Comment:
You have to add this to `V2MappingUpdate` function as well. That is called when updating the model. The other only when you create the index the first time.